### PR TITLE
Material design

### PR DESCRIPTION
**Pull request by mXaln**

### CODE REVIEW

File: src/main/kotlin/org/wa/rceditor/application/Styles.kt [hunk 1]
```diff
@@ -83,12 +109,25 @@ class Styles : Stylesheet() {
...
+
+            label {
+                textFill = c("#000")
+            }
+        }
+        /*listCell {
+            and(selected) {
+                backgroundColor += c("#94008B")
+            }
+        }*/
```
Comment (mbr4477) [inline src/main/kotlin/org/wa/rceditor/application/Styles.kt:131]: Remove commented code?

File: src/main/kotlin/org/wa/rceditor/application/view/fragments/NewProjectDialog.kt [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+import org.wa.rceditor.application.Styles
+import org.wa.rceditor.application.model.ProjectItem
+import org.wa.rceditor.application.model.ProjectItemModel
+import org.wa.rceditor.application.model.SourceItem
+import org.wa.rceditor.application.model.SourceItemModel
+import tornadofx.*
+
+class NewProjectDialog(listView: JFXListView<ProjectItem>, dialog: JFXDialog): VBox() {
+    var listView: JFXListView<ProjectItem> by singleAssign()
+    var dialog: JFXDialog by singleAssign()
```
Comment (mbr4477) [inline src/main/kotlin/org/wa/rceditor/application/view/fragments/NewProjectDialog.kt:18]: Is this dialog being shown anywhere? I only saw that it was being closed in the commitAll function

Comment (mXaln) [reply src/main/kotlin/org/wa/rceditor/application/view/fragments/NewProjectDialog.kt:18]: @mbr4477 it's being shown in here https://github.com/mXaln/RCEditor/blob/material_design/src/main/kotlin/org/wa/rceditor/application/view/MainView.kt#L362

File: src/main/kotlin/org/wa/rceditor/application/Styles.kt (hunk 1 above)
Comment (mXaln) [reply src/main/kotlin/org/wa/rceditor/application/Styles.kt:131]: @mbr4477 I will remove it, I'm still playing with styles

**Pull request merged**